common/NMEA_parser.h

```diff
@@ -0,0 +1,131 @@
+// NMEA message parser
```

Comment:
Would you mind renaming this file to ``nmea_parser.h``, please, just so we can be consistent with other filenames